### ISSUE TITLE
DMT | Make focus management consistent

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#114187]
## Issue Description

While navigating through form 686c-674, most pages focus on the segmented progress bar when navigating between pages; but within the array builder list loop pages, focus is on the page header (H3). Because of a recent collaboration cycle ticket bringing this to light as a non-launch blocking violation of WCAG 3.2.3 (Consistent Navigation,  level AA), we should implement changes to:
- Ensure that every page has a unique H3
- Update focus management to target the page H3

## Tasks
- [x] Check that every page has a unique H3
- [x] Coordinate with design & content if a page is missing a header, or does not contain unique content
- [x] Set up form to focus on the H3 of every page, except the intro, review & submit, and confirmation pages
- [x] Update unit tests as needed

## Acceptance Criteria
- [x] Focus management is consistent throughout the pages within the form (except intro, review, and confirmation pages)
- [x] Tests have been updated and are all passing
